### PR TITLE
[mergebot] rebase + merge

### PR DESCRIPTION
**Pull request by clee2000**
adds flag to rebase and merge with one command by just running the tryrebase.py script before running the trymerge.py script

testing on https://github.com/clee2000/random-testing/pull/19

corresponding test-infra change: https://github.com/pytorch/test-infra/pull/712

### CODE REVIEW

File: .github/workflows/trymerge.yml [hunk 1]
```diff
@@ -38,8 +38,13 @@ jobs:
           ON_GREEN: ${{ github.event.client_payload.on_green}}
           LAND_CHECKS: ${{ github.event.client_payload.land_checks }}
           COMMENT_ID: ${{ github.event.client_payload.comment_id }}
+          REBASE: ${{ github.event.client_payload.rebase }}
         run: |
           set -ex
+          if [ -n "${REBASE}" ]; then
+            python3 .github/scripts/tryrebase.py "${PR_NUM}"
```
Comment (huydhn) [inline .github/workflows/trymerge.yml:45]: From https://github.com/pytorch/test-infra/pull/712: does it make sense to have REBASE contain the branch name, so we can run `python3 .github/scripts/tryrebase.py "${PR_NUM}" --branch "${REBASE}"`, with a default to master?

Comment (clee2000) [reply .github/workflows/trymerge.yml]: done, it uses master as the default if -r is used, and which ever branch is specified if -r < branch > is used

**Review by janeyx99 — APPROVED** [review]

**Review by huydhn — APPROVED** [review]
It's nicely done!

**Pull request closed without merging**